musicname/ze_ffvii_temple_ancient_v3_3.cfg

```
@@ -0,0 +1,28 @@
+"music"
+{
+    "end1.mp3"                      "FFVII OST - J-E-N-O-V-A"
+    "end2.mp3"                      "FFVII: Crisis Core OST - CRISIS CORE Theme"
+    "waprm.mp3"                     "FFVII OST - Prelude"
+    "stage2n.mp3"                   "FFVII: Crisis Core OST - The Shrouded Village"
+    "stage2n1.mp3"                  "FFVII: Crisis Core OST - Pipes and Steel"
+    "1exnachalo.mp3"                "Epic Score - Enraged"
+    "ex1end1.mp3"                   "Audiomachine - Death Mask"
+    "ex1end2.mp3"                   "Gothic Storm - Rage"
+    "ex1nacp1.mp3"                  "Epic Score - Rage from Within"
+    "ex1nacp2.mp3"                  "Epic Score - Drop the Hammer"
+    "ex1nacp3.mp3"                  "Epic Score - Prison Riot"
+    "end_world.mp3"                 "Cliff Lin - Total Annihilation"
+    "ffvii_back.mp3"                "FVII: Advent Children OST - Those Who Fight"
+    "ffvii_boss_batle_remix.mp3"    "FFX OST - Battle Theme"
+    "ffvii_triface_battle.mp3"      "Fairy Tale OST - Tai Ma Gekisen"
+    "pendulum_island.mp3"           "Pendulum - Island"
+    "pendulum_painkillerend.mp3"    "Pendulum - Painkiller"
+    "showdown.mp3"                  "Pendudlum - Showdown"
+    "the_qemists_anger.mp3"         "The Qemists - Anger (feat. Kenta Koie)"
```

Not sure about how we want to display featured artists. I personally want to stick with using `ft.` instead of `feat.`
```suggestion
    "the_qemists_anger.mp3"         "The Qemists - Anger (ft. Kenta Koie)"
```